Models/Library.cs

```diff
@@ -7,27 +7,36 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using System.Xml.XPath;
+using Newtonsoft.Json;
+using EnlightenMAUI.Common;
 
 namespace EnlightenMAUI.Models
 {
     internal class Library
     {
-        Dictionary<string, Measurement> library;
-        Dictionary<string, double[]> originalRaws;
-        Dictionary<string, double[]> originalDarks;
+        Dictionary<string, Measurement> library = new Dictionary<string, Measurement>();
+        Dictionary<string, double[]> originalRaws = new Dictionary<string, double[]>();
+        Dictionary<string, double[]> originalDarks = new Dictionary<string, double[]>();
 
         Logger logger = Logger.getInstance();
         List<Task> loaders = new List<Task>();
 
+        Wavecal wavecal;
+
         public Library(string root, Spectrometer spec)
         {
             logger.debug($"instantiating Library from {root}");
             AssetManager assets = Platform.AppContext.Assets;
-            string[] assetP = assets.List(root);
+
+            string[] assetP = assets.List("libraries/SiG-785");
```

Comment:
So we don't _want_ to get acetaminophen here?

Reply:
I'm not sure I understand...this constructor is called using the OEM folder in the scope view model. The string literal was from when I was debugging and should/will be removed